src/driver/express/ExpressDriver.ts

```diff
@@ -228,19 +229,24 @@ export class ExpressDriver extends BaseDriver implements Driver {
         }
 
         // set http status code
-        if (action.undefinedResultCode && result === undefined) {
-            if (action.undefinedResultCode instanceof Function)
-                throw new (action.undefinedResultCode as any)(options);
-
-            options.response.status(action.undefinedResultCode);
-
-        } else if (action.nullResultCode && result === null) {
-            if (action.nullResultCode instanceof Function)
-                throw new (action.nullResultCode as any)(options);
-
-            options.response.status(action.nullResultCode);
-
-        } else if (action.successHttpCode) {
+        if (result === undefined) {
+            if (action.undefinedResultCode) {
+                if (action.undefinedResultCode instanceof Function) {
```

Comment:
I know it has been before, but yet this is so bad naming. I would expect it to be number only. We should do some refactoring later.